transform/models/marts/dim_users.sql

```diff
@@ -0,0 +1,3 @@
+Select distinct 
```

Comment:
Style: Make sure that the casing is consistent (e.g. uppercase for SQL statements like you opted for in other files)